src/main/scala/edu/twitter/SentimentAnalyzer.scala

```diff
@@ -32,15 +33,27 @@ object SentimentAnalyzer extends App {
   val modelService = new ModelService(new GradientBoostingBuilder(sc))
   modelService.start()
 
+  val indexHandler = new IndexHandler
+  val result = indexHandler.create("twitter", "sentiment")
```

Comment:
Could you please change the name of `result` to something like `indexCreationResult`